### PR TITLE
1.19/1.19

### PR DESCRIPTION
**Pull request by wagyourtail**
leijurv, give me a branch :stuck_out_tongue: 

todo:
- [x] reverify building
- [x] wait for forge
- [x] refreeze loom version

### CODE REVIEW

File: buildSrc/src/main/java/baritone/gradle/task/BaritoneGradleTask.java [hunk 1]
```diff
@@ -49,8 +49,8 @@ class BaritoneGradleTask extends DefaultTask {
             ARTIFACT_STANDALONE         = "%s-standalone-%s.jar",
             ARTIFACT_FORGE_UNOPTIMIZED  = "%s-forge-unoptimized-%s.jar",
             ARTIFACT_FORGE_API          = "%s-forge-api-%s.jar",
-            ARTIFACT_FORGE_STANDALONE   = "%s-forge-standalone-%s.jar",
-            ARTIFACT_FABRIC_UNOPTIMIZED = "%s-fabric-standalone-%s.jar",
```
Comment (scorbett123) [inline buildSrc/src/main/java/baritone/gradle/task/BaritoneGradleTask.java]: Why have you removed forge and fabric from the names here?

Comment (wagyourtail) [reply buildSrc/src/main/java/baritone/gradle/task/BaritoneGradleTask.java]: o but I thought I undid that as I handled it differently below, good catch

**Comment by wagyourtail** [comment]
I'm using a jank solution to fix tweaker version, but I froze the loom version, so it shouldn't break on me...

image

**Comment by robigan** [comment]
@wagyourtail Out of curiosity, is the current state at the time of writing of this 1.19 branch usable enough for mining on Fabric 1.19? (Or even standalone?) I just need to clear a large area out asap lol

**Comment by robigan** [comment]
Only issue I found is that I get NaN when invoking eta command whilst clearing an area out. That normal?

**Comment by ZacSharp** [comment]
Yes, that's the explicit default in case there is no ETA to be shown.

**Comment by 42Mockingjay** [comment]
> @wagyourtail Out of curiosity, is the current state at the time of writing of this 1.19 branch usable enough for mining on Fabric 1.19? (Or even standalone?) I just need to clear a large area out asap lol
> 
> Edit: Nevermind, compiled it myself and it works pretty smoothly

Can you send the compiled version to me? I don't know how to compile it.

**Comment by ZacSharp** [comment]
I'd suggest having a look at the [CI builds](https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml?query=event%3Apull_request+branch%3A1.19%2F1.19+is%3Asuccess). Just select the newest one and download the artifacts at the bottom of the page.

**Comment by 42Mockingjay** [comment]
> I'd suggest having a look at the [CI builds](https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml?query=event%3Apull_request+branch%3A1.19%2F1.19+is%3Asuccess). Just select the newest one and download the artifacts at the bottom of the page.

thanks

**Comment by robigan** [comment]
> @robigan
> 
> > @wagyourtail Out of curiosity, is the current state at the time of writing of this 1.19 branch usable enough for mining on Fabric 1.19? (Or even standalone?) I just need to clear a large area out asap lol
> > Edit: Nevermind, compiled it myself and it works pretty smoothly
> 
> Can you send the compiled version to me? I don't know how to compile it.

Yes I can. But given that I am an internet stranger, if I were you, I wouldn't trust me.
Anyways here's the files;
https://cdn.discordapp.com/attachments/985630869265973308/986706760918261760/baritone-fabric-api-1.9.0-local.jar
https://cdn.discordapp.com/attachments/985630869265973308/986706761140568105/baritone-fabric-standalone-1.9.0-local.jar
https://cdn.discordapp.com/attachments/985630869265973308/986706761371250708/baritone-fabric-unoptimized-1.9.0-local.jar

**Comment by olucaslab** [comment]
Any way to build the .jar of this new version?

**Comment by NatoBoram** [comment]
@olucaslab As usual, check the actions: https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml

**Pull request merged**